Recipes/TrainingDemo.recipe.json

```diff
@@ -65,6 +65,108 @@
           "Description": "The widgets in this layer are only displayed on the homepage."
         }
       ]
+    },
```

Comment:
Ide a worklfow-k elé tehetsz pár mondatot, ami szemlélteti, hogy mit látunk itt a recipe-ben. Mit szólsz egy ilyenhez?

// Workflows
// The User Login logger and the Person Page created logger workflows represent how you can create Workflows in Orchard Core that adds entries to the log when a user logs in or when someone creates a new Person Page content item.

Persze ha ez nem fér ki 120 sorba, akkor több sorban írd be, ahogy például a Modules-nál is kommentezve van.